triscord/__init__.py

```diff
@@ -92,6 +92,9 @@ def main(config_path, persist_path, debug=False):
         for action in feed_actions:
             message = feed.format_action(action)
             if message:
+                for trello_user, alias in settings.CONFIG.items('UserAlias'):
+                    message = message.replace(trello_user, alias)
```

Comment:
Doing a big simple text substitution like this one is bound to have unwanted side effects.

What if you have your own website that goes `enricoghidoni.com` and you link it in a trello card? It'll become `Henrik.com`.

Reply:
Would you rather have it implemented with a regex match?

Reply:
How would that fix the problem ?